docs/index.rst

```diff
@@ -39,46 +39,14 @@ need to test against a specific revision:
 Getting Started
 ---------------
 
-Once installed, you can quickly bootstrap a new Taskgraph setup by running the
-following command within an existing repository:
+Once installed, check out the :doc:`/tutorials/getting-started` tutorial to get
```

Comment:
nit: "check out" seems too...colloquial? I suggest "read", "view", or "see": "Once installed, read the tutorial to get a task up and ..."